Makefile

```diff
@@ -12,14 +12,27 @@ IMAGE_TAG ?= latest
 OPERATOR_IMAGE ?= cluster-network-addons-operator
 REGISTRY_IMAGE ?= cluster-network-addons-registry
 
+fmt: goimports whitespace
+
+goimports:
+	goimports -w ./pkg ./cmd
+
+whitespace:
+	./hack/whitespace.sh --fix
+
+check: whitespace-check vet goimports-check generate-api-check test
+
+whitespace-check:
+	./hack/whitespace.sh
+
 vet:
```

Comment:
Create a fake file to track timestamps we have to also remove it from .PHONY since it really creates a file

vet: $(cmd_sources) $(pkg_sources)
go vet ./pkg/... ./cmd/...
touch $@

Reply:
Done